server/helpers/planner_helper/planner_helper_data_types.py

```diff
@@ -164,28 +179,6 @@ def check_for_none(self) -> PlanDisambiguatorOutput:
         return self
 
 
```

Comment:
Merged this PlanningTask with the other one and moved the model validator to the other class. We no longer need extra checks at the start of each endpoint then, the validator will throw an exception if the input is not valid.